docs/Summary.md

```diff
@@ -18,8 +18,8 @@
 | [PX1014](diagnostics/PX1014.md) | A DAC field must have a nullable type. | Error   | Available |
 | [PX1015](diagnostics/PX1015.md) | For a BQL statement that contains parameters, the number of arguments of a `Select` method is different from the number of parameters. | Warning (Level 1: Significant) | Unavailable |
 | [PX1018](diagnostics/PX1018.md) | The graph with the specified primary view type parameter doesn't contain the primary view of the specified type. | Error | Unavailable |
-| [PX1021](diagnostics/PX1021.md) | The DAC property field has a type that is not compatible with the field attribute assigned to this property. | Error | Available |
-| [PX1023](diagnostics/PX1023.md) | The DAC property is marked with multiple field attributes. | Error | Available |
+| [PX1021](diagnostics/PX1021.md) | The DAC property field has a type that is not match with the field attribute assigned to this property. | Error | Available |
+| [PX1023](diagnostics/PX1023.md) | The DAC property is marked with multiple type attributes. | Error | Available |
```

Comment:
There might not only type attributes, but also some special attribute like `PXDBCalced`.
This item is mentioned in ATR-406.